src/Performance/PerformanceConnector.ts

```diff
@@ -0,0 +1,170 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Mateusz Bernacik
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+
+import External from '../Shared/External';
+import PAUSConnector from '../Shared/PAUSConnector';
+import { TestConverter } from './Converters';
+import PerformanceOptions, {
+    PerformanceRequestPayload,
+    PerformanceConverterOptions
+} from './PerformanceOptions';
+import type MorningstarConverter from '../Shared/MorningstarConverter';
+import type PerformanceJSON from './PerformanceJSON';
+
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+
+/* *
+ *
+ *  Interfaces
+ *
+ * */
+
+export interface PerformanceConverter extends MorningstarConverter {
+    parse(options: PerformanceConverterOptions): void;
+}
+
+/* *
+ *
+ *  Constants
+ *
+ * */
+
+export const DATA_TABLES = [
+    { key: 'TestConverter' }
+];
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class PerformanceConnector extends PAUSConnector {
+
+
+    /* *
+     *
+     *  Constructors
+     *
+     * */
+
+
+    public constructor (
+        options: PerformanceOptions
+    ) {
+        super(options, DATA_TABLES);
+
+        this.options = options;
+        this.url = `/portfolioanalysis/v1/performance?langcult=${this.options.langcult || 'en-US'}`;
```

Comment:
`langcult` should be implemented on PAUSConnector (since options is available to all connectors)